src/Manager.js

```diff
@@ -406,6 +410,9 @@ class GiveawaysManager extends EventEmitter {
                 await this.editGiveaway(giveaway.messageID, giveaway.data);
                 return;
             }
+            if (giveaway.options.isDrop == true && giveaway.message.reactions.cache.get(giveaway.reaction).count - 1 >= giveaway.winnerCount) {
```

Comment:
```suggestion
            if (giveaway.options.isDrop && giveaway.message.reactions.cache.get(giveaway.reaction).count - 1 >= giveaway.winnerCount) {
```
Not needed because you already added that it makes sure that it only is a boolean

Reply:
But what if someone set's it to false tho?

Reply:
Oh yes true thanks for that correction.